README.md

```diff
@@ -0,0 +1,128 @@
+# Yagna-Wasmtime integration
+![Continuous integration](https://github.com/golemfactory/ya-runtime-wasi/workflows/Continuous%20integration/badge.svg)
+
+`ya-runtime-wasi` is a [`Yagna`] plugin that allows the provider to execute WASI modules
+in a safe, sandboxed way. Typically, you will use this crate as part of your Yagna provider
+installation. However, it is also possible to use the integration standalone to execute
+zipped WASI modules according to some included manifest file.
+
+[`Yagna`]: https://github.com/golemfactory/yagna
+
+## Building
+
+Building the project is very straightforward:
+
+```
+cargo build
+```
+
+If you decide to make some tweaks and would like to test if everything still behaves
+as expected, you can trigger included end-to-end integration tests like so:
+
+```
+cargo test --features integration-tests
+```
+
+Note that running the end-to-end tests requires you to have `wasm32-wasi` target installed.
+
+## Running
+
+### As part of Yagna
+
+This step is explained in [Yagna's general tutorial].
+
+[Yagna's general tutorial]: https://github.com/golemfactory/yagna/tree/master/agent/provider
+
+### Standalone
+
+Running standalone is pretty simple. For this, you'll want to use a Wasm module which performs
+some input from the host, does some computations, and outputs the results to a file on the host.
+To keep everything simple, we'll assume you use [`rust-wasi-tutorial`]. Clone the repo, and
+build the project:
+
+```
+git clone https://github.com/kubkon/rust-wasi-tutorial.git
```

Comment:
is it on purpose?

```suggestion
git clone https://github.com/golemfactory/rust-wasi-tutorial.git
```

Reply:
Yep. I thought the example package was using my tutorial. We don't have repo under this name in golemfactory. Besides, I don't think there is any point in forking it. It's just a tutorial.